x-pack/plugins/fleet/public/applications/fleet/hooks/use_request/enrollment_api_keys.ts

```diff
@@ -65,3 +67,11 @@ export function useGetEnrollmentAPIKeys(
     ...options,
   });
 }
+
+export function sendCreateEnrollmentAPIKey(body: PostEnrollmentAPIKeyRequest['body']) {
```

Comment:
Not blocking: do you we could use this in the new_enrollment_key_modal file too? instead of the sendRequest directly

Reply:
👍🏻 Thanks for the reminder. I had that in mind but forgot to come back to it

Reply:
Did this in https://github.com/elastic/kibana/pull/100599/commits/025e155091ba4a48d4d7b429e23f343c66ccd932#diff-85b2827017906097a0a9973c5eec975cc3f470cb5ab8549133f2d6dfecce12b9